src/main/java/frc/robot/RobotContainer.java

```diff
@@ -123,6 +131,7 @@ public RobotContainer() {
 
     } else {
       drivetrain = new Drivetrain(new DrivetrainIO() {});
+      intake = new Intake(new IntakeIOTalonFX());
```

Comment:
This else case creates subsystems without a hardware implementation. To do this we create an anonymous class that implements the interface:

```
intake = new Intake(new IntakeIO() {});
```